src/main/kotlin/uk/gov/justice/digital/assessments/api/ErrorResponseOffender.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.digital.assessments.api
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.v3.oas.annotations.media.Schema
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+data class ErrorResponseOffender(
+  @Schema(required = true, description = "Fullname of the offender", example = "Ray Arnold")
+  val name: String? = null,
```

Comment:
is the full name GDPR compliant? can we include things like this in logs/errors? might worth double checking

Reply:
Good point - we require the context to be returned to the frontend in order to display as part of the error message. Will do a pass to ensure this isn't passed in to the logs

Reply:
Removed this code - agreed to split this out as a separate ticket, we may be able to get this information back in the OASys error response